docs/LV_client/utilities/mono_get_binary_file.md

```diff
@@ -0,0 +1,40 @@
+# mono_get_binary_file.vi
+
+<p class="img_container">
+<img class="lg_img" src="../mono_get_binary_file.png"/>
+</p>
+
+### Description
+
+Obtain the reference to the binary file to read based on the name of the sensor type and the custom name assigned by the user. 
+
+For technical support contact us at <b>support@monodrive.io</b>  
+
+### Inputs
+
+- **Sensor configuration:**  Path to the configuration file created by the mono_logger.vi
+ 
+
+- **error in (Error Cluster):** Accepts error information wired from previously called VIs. This information can be used to decide if any functionality should be bypassed in the event of errors from other VIs. 
+
+### Outputs
+
+- **Channels:**  If the sensor configurated was a camera, this VI will
+extracted the number of channels from the the configuration
+file
+ 
+
+- **Type:**  Type of sensor obtained from the sensor configuration
+ 
+
+- **Stream dimensions:**  If the type of sensor is a camera returned the stream
```

Comment:
Maybe try:  "If the sensor type is camera, the stream dimensions will be returned in order to parse the image correctly